### ISSUE TITLE
TOTP title is "WordPress.org Forums"

### ISSUE THREAD

**Issue by dd32** [WordPress/wporg-two-factor#69]
Currently the TOTP issuer is listed as `WordPress.org Forums`, as that's the site that the TOTP auth setup is presented on.

We should change this to either be `WordPress.org`, `WordPress.org Login` or something similar as it's not forums-specific.

**Comment by iandunn** [comment]
Huh, I thought I [fixed that already](https://github.com/WordPress/wporg-two-factor/issues/48) 🤔

**Comment by dd32** [comment]
Hmm.. you did, but TOTP isn't using a consistent filter.

There's two fields, `label` which should be `"$issuer: $accountName"`, and `issuer` which should be `"$issuer"`.

That filter only changes the label, it doesn't change the `issuer`, and as a result the TOTP QR no longer follows this google-authenticator line:

https://github.com/google/google-authenticator/wiki/Key-Uri-Format#issuer (emphasis mine)
> STRONGLY RECOMMENDED: The issuer parameter is a string value indicating the provider or service this account is associated with, URL-encoded according to [RFC 3986](http://tools.ietf.org/html/rfc3986). If the issuer parameter is absent, issuer information may be taken from the issuer prefix of the label. **If both issuer parameter and issuer label prefix are present, they should be equal.**

The QR generated is something like this:
```
otpauth://totp/WordPress.org:dd32?secret=KEYKEYKEY&issuer=WordPress.org Forums
```

Basically, the `two_factor_totp_title` filter should only be used to change the component after the colon.

A new filter of `two_factor_totp_issuer` which controls the input into the second filter would probably be the most correct otpion here.

**Comment by iandunn** [comment]
Ah, thanks!
